src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -95,7 +95,7 @@ private function populateEntrypointProperties(string $resourceClass, ResourceMet
                 '@type' => 'hydra:Link',
                 'domain' => '#Entrypoint',
                 'rdfs:label' => "The collection of $shortName resources",
-                'range' => 'hydra:PagedCollection',
+                'range' => 'hydra:Collection',
```

Comment:
Although POST is a collection operation, can the range be considered correct for it?